projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.spec.ts

```diff
@@ -91,6 +91,13 @@ describe('CommentComponent', () => {
     expect(component.fullComment).toEqual(mockComment.content);
   });
 
+  it('should get comment', () => {
+    component.comment = {...mockComment};
+
+    expect(component._comment).toEqual({...mockComment});
+    expect(component.comment).toEqual({...mockComment});
```

Comment:
I don't get this, you just set it a couple of lines above, so obviously it's gonna be the same